PasswordVerifier.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace PasswordVerifier
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter your password: ");
+            string password;
```

Comment:
Read about [SecureString class](https://docs.microsoft.com/en-us/dotnet/api/system.security.securestring?view=net-6.0), and use it.

Reply:
Done! Resolved on the new commit. Thank you!